docs/source/top_level_elements/genre.rst

```diff
@@ -0,0 +1,163 @@
+genre
+=====
+
+About
+-----
+
+This section describes the different values for `genre` that we currently have in our Islandora repository.
+
+genre[@authority = 'dct']
+-------------------------
+
+We will explore other options for records using `@authority = 'dct'` or uncontrolled vocabularies, e.g. mapping values as `typeOfResource`. Many of the records in Volunteer Voices have multiple `genre`s; e.g. `volvoices:14311 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:14311/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="dct">text</genre>
+    <genre>letter</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:type "text" ;
+        edm:hasType "letter" .
+
+Roughly 300 other records have more than two `genre`s; e.g. `volvoices:11262 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:11262/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre>notated music</genre>
+    <genre authority="dct">still image</genre>
+    <genre>sheet music</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> edm:hasType "notated music", "sheet music" ;
+        dcterms:type "still image" .
+
+genre[@authority = 'aat']
+-------------------------
+
+`genre[@authority = 'aat']` appears in the Archivision collection and uses a `@valueURI` for controlled vocabulary; e.g. `archivision:404 <https://digital.lib.utk.edu/collections/islandora/object/archivision:404/datastream/MODS/view>`_. We will map these as subjects.
+
+.. code-block:: xml
+
+    <genre authority="aat" valueURI="http://vocab.getty.edu/aat/300021140">Renaissance</genre>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/object/1> dcterms:subject <http://vocab.getty.edu/aat/300021140> .
+
+genre[@authority = 'lcsh']
+--------------------------
+
+Used in the Archivision and Charlie Daniel collections; e.g. `cDanielCartoon:455 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon:455/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85040974">Editorial cartoons</genre>
+
+.. code-block:: turtle
+
+    @prefix edm: <http://www.europeana.eu/schemas/edm/> .
+
+    <https://example.org/object/1> edm:hasType <http://id.loc.gov/authorities/subjects/sh85040974> .
+
+and `archivision:1754 <https://digital.lib.utk.edu/collections/islandora/object/archivision:1754/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <genre authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85139020">Twentieth century</genre>
```

Comment:
Values like these might ideally be placed in a temporal subject, but I don't think there is an easy way for us to separate these out (aka - problematic, but does not need to be addressed)